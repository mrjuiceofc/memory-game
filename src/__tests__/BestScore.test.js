import { render, screen } from "@testing-library/react";
import BestScore from "../components/BestScore.js";

describe("BestScore component", () => {
  it("renders BestScore", () => {
    const { container } = render(<BestScore bestScore={5} />);
    expect(container).toMatchSnapshot();
  });

  it("shows best score correctly", () => {
    render(<BestScore bestScore={5} />);
    expect(screen.getByText(/best score: 5/i)).toBeInTheDocument();
  });
});
