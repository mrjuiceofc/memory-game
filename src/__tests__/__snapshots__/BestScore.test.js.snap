// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BestScore component renders BestScore 1`] = `
<div>
  <div
    class="bg-violet-500 text-white font-bold inline-block py-2 w-36 rounded-2xl shadow-md shadow-violet-700"
  >
    best score: 
    5
  </div>
</div>
`;
